lib/irc/BridgedClient.js

```diff
@@ -327,6 +327,7 @@ BridgedClient.prototype.whois = function(nick) {
             resolve({
                 server: self.server,
                 nick: nick,
+                isOp: whois.operator !== undefined,
```

Comment:
Fair enough, apparently the /WHO command might be the way to get a more detailed response.